tool/plugin/lib/plugin.dart

```diff
@@ -316,6 +348,12 @@ Future<int> zip(String directory, String outFile) async {
   return await exec('zip', args, cwd: p.dirname(directory));
 }
 
+String _nextRelease() {
+  var num = RegExp(r'release_(\d+)').matchAsPrefix(lastReleaseName).group(1);
```

Comment:
`num` is a type; I assume this is valid Dart (as it analyzes correctly), but perhaps name the variable something else?

Reply:
Good point. Fixed.